### PR TITLE
Added package odb-pgsql

### PR DESCRIPTION
**Pull request by pretyman**
This package depends on package PostgreSQL, since it is odb support for the PostgreSQL database

### CODE REVIEW

File: cmake/projects/odb-pgsql/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+else()
+  set(HUNTER_CMAKE_PROJECTS_ODB-PGSQL_HUNTER_CMAKE 1)
+endif()
+
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_package(odb)
+hunter_add_package(PostgreSQL)
```
Comment (ruslo) [inline cmake/projects/odb-pgsql/hunter.cmake]: Not needed here since we have them in scheme already

File: cmake/projects/odb-pgsql/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    URL
+    "http://www.codesynthesis.com/download/odb/2.4/libodb-pgsql-2.4.0.tar.bz2"
+    SHA1
+    4628d5e296da01dbaf8658fd402b4f709f30ea2d
+)
+
+hunter_pick_scheme(DEFAULT url_sha1_odb-pgsql_autotools)
+hunter_download(
+    PACKAGE_NAME odb-pgsql
+    PACKAGE_DEPENDS_ON odb PostgreSQL
```
Comment (ruslo) [inline cmake/projects/odb-pgsql/hunter.cmake]: Not needed

File: cmake/projects/odb-pgsql/hunter.cmake (hunk 1 above)
Comment (pretyman) [inline cmake/projects/odb-pgsql/hunter.cmake]: Ok, I used the `odb-sqlite` template, where in the past we've decided to put dependencies in `projects/../hunter.cmake` and on the scheme script. I'll remove these.

File: cmake/projects/odb-pgsql/hunter.cmake (hunk 2 above)
Comment (pretyman) [inline cmake/projects/odb-pgsql/hunter.cmake]: This is from `odb-sqlite` template, where in the past we've decided to use `PACKAGE_DEPENDS_ON`.

**Comment by pretyman** [comment]
I'll open a new pull request with your suggested changes for a clean history

**Pull request closed without merging**